diff -urN jlint-3.1.1/access_desc.cc jlint-modified/access_desc.cc
--- access_desc.cc	2001-01-19 00:48:45.000000000 +0300
+++ access_desc.cc	2010-04-30 11:34:25.000000000 +0400
@@ -1,4 +1,5 @@
 #include "access_desc.hh"
+#include <stdarg.h>
 
 void access_desc::message(int code, ...)
 {
diff -urN jlint-3.1.1/access_desc.hh jlint-modified/access_desc.hh
--- access_desc.hh	2001-01-31 18:49:49.000000000 +0300
+++ access_desc.hh	2010-04-30 11:45:43.000000000 +0400
@@ -1,6 +1,7 @@
 #ifndef ACCESS_DESC_HH
 #define ACCESS_DESC_HH
 
+#include <stdlib.h>
 #include <stdarg.h>
 
 class field_desc;
diff -urN jlint-3.1.1/graph.hh jlint-modified/graph.hh
--- graph.hh	2001-01-31 18:49:49.000000000 +0300
+++ graph.hh	2010-04-30 11:40:30.000000000 +0400
@@ -1,6 +1,7 @@
 #ifndef GRAPH_HH
 #define GRAPH_HH
 
+#include <string.h>
 #include "types.hh"
 #include "method_desc.hh"
 
diff -urN jlint-3.1.1/message_node.hh jlint-modified/message_node.hh
--- message_node.hh	2001-03-16 23:49:19.000000000 +0300
+++ message_node.hh	2010-04-30 11:46:00.000000000 +0400
@@ -1,5 +1,6 @@
 #ifndef MESSAGE_NODE_HH
 #define MESSAGE_NODE_HH
+#include <string.h>
 //#include <string>
 //#include "types.hh" // included by functions.hh
 #include "functions.hh"
diff -urN jlint-3.1.1/utf_string.hh jlint-modified/utf_string.hh
--- utf_string.hh	2003-04-26 18:00:24.000000000 +0400
+++ utf_string.hh	2010-04-30 11:41:07.000000000 +0400
@@ -1,7 +1,7 @@
 #ifndef UTF_STRING_HH
 #define UTF_STRING_HH
 
-#include <string>
+#include <string.h>
 #include <stdio.h>
 #include <assert.h>
 #include "functions.hh"
